### PR TITLE
Update season 1 challenges and add challenges for the rest of the locations

### PR DESCRIPTION
**Pull request by moonysolari**
All challenge data were acquired with the `extract-challenge-data` script with no modifications.

**What works**
- Updated the season 1 challenges with some recently added ones.
- All destinations now have challenges shown in the Challenges tab on the location page.
- All contracts now have challenges shown in the Challenges tab on the planning page and in-game F1 page, including sniper ones.
- Players can complete challenges on sniper maps, and the completion data saves. (Tested with Psycho Daisies in The Last Yardbird)

**What might not work**
- I did not manually test every challenge and confirm that it works.
- Dartmoor Garden Show challenges are not working.

### CODE REVIEW

**Review by AnthonyFuller — CHANGES_REQUESTED** [review]
I'm not even going to point out all of the instances that the Arcade challenges show up.

Elusive Target Arcade is not a feature of Peacock, and I don't think our challenge system can even mark them as completed. I say they should be removed, but this is of course, up for discussion.

**Comment by ColossusChang** [comment]
> Arcade challenges show up

I noticed them as well and wavered between deleting them and keeping them. Seeing that the original `json`s had them, I decided to keep them.

> I don't think our challenge system can even mark them as completed. 

It cannot, but having them there does not interfere with other features either, as far as I know.

> I say they should be removed (until ETA is added to Peacock pending the escalation rewrite), but this is of course, up for discussion.

That I can do, but we will need to add them back sometime anyway... We can wait for another reviewer's take on this.
I personally am fine with either way.

**Comment by moonysolari** [comment]
> Arcade challenges show up

I noticed them as well and wavered between deleting them and keeping them. Seeing that the original `json`s had them, I decided to keep them.

> I don't think our challenge system can even mark them as completed. 

It cannot, but having them there does not interfere with other features either, as far as I know.

> I say they should be removed (until ETA is added to Peacock pending the escalation rewrite), but this is of course, up for discussion.

That I can do, but we will need to add them back sometime anyway... We can wait for another reviewer's take on this.
I personally am fine with either way.

**Comment by AnthonyFuller** [comment]
> Seeing that the original jsons had them, I decided to keep them.

Ah, I did not realise the originals had them too. I thought they had the category but were empty.

**Review by AnthonyFuller — APPROVED** [review]
Putting my previous comment aside, it all LGTM. Pending @RDIL to clear it by merging it.

**Comment by moonysolari** [comment]
Dartmoor Garden Show challenges are not working for some reason. Might be related to #78 but I haven't investigated yet. Can merge first and fix later after #63 is merged.

**Review by RDIL — APPROVED** [review]

**Pull request merged**
